mb/scripts/mb

```diff
@@ -877,12 +884,12 @@ class MirrorDoctor(cmdln.Cmdln):
         for i in range(opts.quietness):
             cmd.append('-q')
 
-        if opts.enable:
-            cmd.append('-e')
```

Comment:
why do you remove this?

Reply:
Mirrors are enabled from python, not need to pass that option to perl